openadapt/window/_windows.py

```diff
@@ -133,7 +130,15 @@ def get_element_properties(element):
     return properties
 
 
-def dictify_rect(rect):
+def dictify_rect(rect: Any) -> dict:
```

Comment:
could you please change this to pywinauto.win32structures.RECT?

Reply:
addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/6159a69dd7340dd520bd020a57101dfd9d444c9f